specification/servicebus/resource-manager/readme.typescript.md

```diff
@@ -0,0 +1,12 @@
+## TypeScript
+
+These settings apply only when `--typescript` is specified on the command line.
+Please also specify `--typescript-sdks-folder=<path to root folder of your azure-sdk-for-js clone>`.
+
+``` yaml $(typescript)
+typescript:
+  azure-arm: true
+  package-name: "@azure/arm-sb"
```

Comment:
@amarzavery Can we please rename this package to arm-servicebus? This is the only package that decided to use its initials instead of its name, and it bothers me every time I see it.